package.json

```diff
@@ -18,13 +18,16 @@
     "hot": "NODE_ENV=development webpack-dev-server --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
     "production": "NODE_ENV=production webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
     "install": "NODE_ENV=production npm run production",
-    "postinstall": "npm run fetch_blogs && npm run fetch_projects",
-    "fetch_projects": "node node_modules/eclipsefdn-hugo-solstice-theme/bin/projects/index.js",
+    "postinstall": "npm run fetch_blogs && npm run fetch_projects && npm run yaml2json_projects",
+    "fetch_projects": "node node_modules/eclipsefdn-hugo-solstice-theme/bin/projects/index.js -l data -u https://projects.eclipse.org/api/projects?working_group=cloud-development-tools",
```

Comment:
In this case, we are already fetching the data from API, not the static JSON file? It is only using the static JSON file if the API is not working if I understand correctly. 🤔

Reply:
What do you mean? Just to clear it up, the fetch_projects script fetches the live data at build time, creates a local copy, and the JS references that file. Looking at the JS there is no fallback case, it always retrieves this pre-fetched data, and never the live data. 

The reason that this script was initially created was so that Hugo could reference projects at build time (as it is a static generator). While this is not needed with a react component, it would accelerate development as you wouldn't need to code for retrieval of paginated data initially. I'd recommend not replacing this until the rest of the component is fully finished.